Sync Network release branch with master

anton-evseev opened the pull request:
I see there were some changes to pipelines in master, so we'll need to sync that with our release branch
<br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

nschonni committed:
chore: jsonfmt sql

sergey-shandar committed:
Update preproduction-azure-pipelines.yml (#6589)

* Update preproduction-azure-pipelines.yml

* Update preproduction-azure-pipelines.yml

xiaoyingQQ committed:
support teradata dataset in adf (#6568)

* support teradata dataset in adf

* remove Teradata linked service required property

nschonni committed:
chore: jsonfmt arm-compute (#6639)

davidzhaoyue committed:
[Datafactory] Add three new connectors (#6607)

* [Datafactory] Add three new connectors

* Add Informix into custom-words.txt

Commit:
[Hub Generated] Review request for Microsoft.Resources to add version stable/2019-06-01 (#6610)

* Adds base for updating Microsoft.Resources from version stable/2018-06-01 to version 2019-06-01

* Updates readme

* Updates API version in new specs and examples

* Adding managedByTenants to subscription

Adding managedByTenants property for GET subscription and GET subscriptions calls.

* Fixing ManagedByTenant

Putting tenantId in 'properties' to fix errors

* Adding description to satisfy warning

Adding description for ManagedByTenant to satisfy a warning.

* Adding GetSubscriptions example

* Adding GetSubscription.json example

* Update GetSubscriptions.json

* Adding examples

* Update GetSubscription.json

* Update GetSubscriptions.json

* Update subscriptions.json

* Update GetSubscriptions.json

* Update GetSubscription.json

* Update GetSubscription.json

* Update GetSubscriptions.json

* Update GetSubscription.json

* Update GetSubscriptions.json

* Update GetSubscriptions.json

celikcigdem committed:
Dev/ccelik/dedicatedhostswagger (#6612)

* Update compute.json

* Added DedicatedHostGroup and DedicateHost operations

* Fixed dedicated host and group examples

* Removed extra comma

* Fixed model validation errors

* Added more examples

* Modified examples

* fixed PlatformFaultDomain type

* parameter name fixes

* Suppressed errors on DedicatedHostGroupUpdate and DedicatedHostUpdate

* Update compute.json

* Update compute.json

* Merge from upstream/master

* Update compute.json

Documentation changes and marked required fields

* Update GetADedicatedHost.json

* remove .orig files

* Delete compute.json.orig

* Use min max

* attempt to fix model validation

* fix model validation

* Removed availableVCpus

* Update GetADedicatedHost.json

nschonni committed:
chore: jsonfmt cognitiveservices (#6650)

zfchen95 committed:
[Bug fix] Remove read-only constraint for management policy, and change properties types from integer to number (#6551)

* [Bug fix] Remove read-only constraint for management policy

* [Bug fix] Change daysAfterCreationGreaterThan and daysAfterModificationGreaterThan to float type

* change type to number

* Modify description, remove integer

* Fix existing build errors in Storage swagger

nschonni committed:
chore: jsonfmt compute (#6649)

Commit:
Update specificationRepositoryConfiguration.json (#6651)

HaoQian-MS committed:
SSIS File System Support (#6548)

iscai-msft committed:
Alertsmanagement: updated java tags and added java readme (#6498)

* generates java code into azure-mgmt-alertsmanagement folder

* updated alertsmanagement java tags

shurd committed:
fixing Cosmos account keys response (#6587)

nschonni committed:
chore: Cleanup package.json dependencies (#6619)

* chore: Remove unused dependencies

* chore: Add dependiens for multiapi.ts

jerryc3 committed:
Add new API version 2019-07-01 for SIG swagger spec (#6400)

* copy over gallery spec from 2019-03-01 api version

* remove PublishingProfile.source, make GalleryImageVersionStorageProfile writable

* fix json

* remove required source

* add 2019-07-01 to Go SDK

* update examples with correct api version and add 1 more example

* add reference to example{

* Add HyperVGeneration field

* Make hyperVGeneration field same as DiskRPs

parisa-naeimi committed:
Introducing new version API 2019-05-01-preview with adding a new filter for usageDetails called publisherType (#6575)

* Adds base for updating Microsoft.Consumption from version preview/2019-04-01-preview to version 2019-05-01-preview

* Updates readme

* Updates API version in new specs and examples

* 1. Added publisherType filter for UsageDetails API in Filter section
2. Added filter section missing in UsageDetailsDownload and also added language parameter missing from specification/consumption/resource-manager/Microsoft.Consumption/preview/2019-05-01-preview/consumption.json
3. Updated Usage Detail Download example to have language parameter

* Added x-ms-parameter-location to language and formatted consumption.json based on the comments

* Removed language from Usage Detail Download based on Reviewer' s comment to be compatible with ARM

leizhaoPS committed:
[READY TO MERGE] Add missing usages api optional parameter (#6656)

* add show usages detial parameter

* add related mark for optioanl query parameter

* remove doller sign from the custome parameter

shleiAmy committed:
update attestation swagger (#6655)

* update attestation swagger

* Move attestation API under a preview folder

* fix the error response

* remove the pattern validation from swaggle since the validation already appears on the server

nschonni committed:
chore: jsonfmt security (#6625)

nschonni committed:
chore: jsonfmt resources (#6648)

roherzbe committed:
Add data connector kinds AzureAdvancedThreatProtection and MicrosoftDefenderAdvancedThreatProtection. (#6614)

* New data connector kinds (MicrosoftDefenderAdvancedThreatProtection & AzureAdvancedThreatProtection) and added discovery logs to MicrosoftCloudAppSecurity.

* Use AlertsDataTypeOfDataConnector

* Fix spell check errors

* Spell check fixes

* Add AATP and MDATP to custom-words

* Using AATP & MDATP after updated custom-words

* Fix mistake

nschonni committed:
chore: Update quickstart-templates from root (#6647)

Copied over the formatted json files from arm-compute at the root

TinaHu1 committed:
[DataFactory]Add Binary Dataset in swagger (#6631)

* Add binary dataset in swagger

* add binary sink

* add type for binary type properties

jhendrixMSFT committed:
add network package-2019-06 to Go SDK (#6671)

jhendrixMSFT committed:
fix some readme.go.md files (#6669)

jhendrixMSFT committed:
fix some readme.go.md files (#6668)

swetha-anantharajaiah committed:
Adding specification for 2019-07-01-preview version for Peering. (#6630)

* adding specification for 2019-07-01-preview version for Peering.

* changing the parameter name

* fixing errors

* fixing validation errors

* updating readme.md to add new version

* changing the name in peeringservice

nschonni committed:
chore: jsonfmt peering (#6678)

nschonni committed:
fix: cognitive services create example (#6638)

Empty schema/body for create action

joerage committed:
[Hub Generated] Review request for Microsoft.DevTestLab to add version stable/2018-09-15 (#6623)

* Modification of PATCH description, plus minor fixes

* fix

* Update specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/DTL.json

Co-Authored-By: Nick Schonning <nschonni@gmail.com>

bfjelds committed:
add winiotsvcs stable api (#6642)

wenbof committed:
[DataFactory]Add AzurePostgreSql Sink and Oracle/SapTable/Netezza Partitions and Several Relational sets (#6617)

* [DataFactory]Add AzurePostgreSql Sink and Oracle/SapTable/Netezza Partitoins and dedicated datasets for relational

* Fix typo.

* update SapBwCube

* fix issue brought by merging from azure branch

* remove SapBwCube and Sybase for investigation

* will postpone SapBwSource in another coming PR.

* refine case of Oracle in description

Commit:
adding package 2019-06 to python multipackage build (#6677)

nschonni committed:
chore: jsonfmt windowsiot (#6684)

zyxyoshine committed:
[Azure Stack] Update Volume and Drive API for Fabric Admin (#6666)

* copy spec content from previous version

* Update Volume and Drive API for Azure Stack Fabric Admin to 2019-05-01

* add the missing word

nschonni committed:
fix: adhybridhealthservice model validation issues (#6414)

* fix: addsServices_update example

- service is an object not an array
- casing on "notificationEmailEnabledForGlobalAdmins"

* fix: addsServicesUserPreference_add example

- setting is an object rather than an array

* fix: configuration_update example

- tenant is an object rather than array

* fix: services_addAlertFeedback example

- alertFeedback is object instead of array
- feeback -> feedback

* fix: serviceMembers_getConnectorMetadata example

- Missing required parameter "metricName"
- flatten value in response

* fix: alerts_listAddsAlerts example

- additionalInformation is an array value

* fix: addsServices_listServerAlerts example

- additionalInformation is an array type

Commit:
Add fork repository configuration (#6694)

nschonni committed:
chore: Sort custom-words.txt (#6698)

Alpha isn't required, but avoids merge conflicts what putting them at the end causes.

tomersh26 committed:
Introduce ADX Command (#6667)

* Introduce ADX Command

* Use full ADX name

tushar-bhargava-ms committed:
IMDS: Adding Swagger spec for 2019-04-30 version. (#6572)

* Adding missing Swagger spec.

* Updated README.md with reference to added directory.

jjbfour committed:
[Hub Generated] Review request for Microsoft.Portal to add version preview/2019-01-01-preview (#6643)

* Adds base for updating Microsoft.Portal from version preview/2015-08-01-preview to version 2019-01-01-preview

* Updates readme

* Updates API version in new specs and examples

* Fix readmes for new api version

nschonni committed:
fix: azsadmin character encoding (#6685)

nschonni committed:
fix: datafactory character encoding (#6686)

nschonni committed:
chore: jsonfmt devtestlabs (#6705)

iscai-msft committed:
added iothub java readme (#5993)

solankisamir committed:
remove id validation (#6703)

nschonni committed:
fix: managementgroups model validation (#6697)

- Add "body" to response
- createGroupRequest -> createManagementGroupRequest

mabhard committed:
removing non-ascii character (#6710)

ruowan committed:
update preproduction pipeline for testing avocado@0.5.0 (#6646)

rokulka committed:
Updated QnAMaker  Runtime API headers descriptions (#6559)

* Updated QnAMaker  Runtime API headers descriptions

* Update swagger title

* removed content-type header

* Common tag and title for Runtime
Undo Authorization header change

* Update title

* update readme

* update readme

ruowan committed:
update package '@azure/rest-api-specs-scripts' 0.4.0 (#6722)

* update package '@azure/rest-api-specs-scripts' 0.4.0

* remove '@azure/rest-api-specs-scripts' related job in preproduction pipeline.

* fix

PakDLiu committed:
Added Keyvault and MSI parameters to create SAP Monitor (#6711)

* Added Keyvault and MSI parameters to create SAP Monitor

* Camel case for new fields

* Fixed casing

tomersh26 committed:
Change type name and add timeout property (#6717)

Commit:
Update azure-sdk-for-go to test in production (#6728)

enjaimes committed:
Add maxComplexObjectsInCollectionsPerDocument to ServiceStats limits (#6721)

Commit:
Change python integration repository to fork (#6729)

juniwang committed:
Add new property for CORS settings (#6478)

nschonni committed:
chore: jsonfmt datafactory (#6654)

nschonni committed:
chore: jsonfmt signalr (#6735)

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-go

azuresdkci commented:
Can one of the admins verify this patch?

number213 commented:
None of `ModelValidation` and `semantic` errors are related to Network, seems like those are existing errors from `master`

nschonni committed:
fix: datafactory model validation (#6733)

Remove header/null body for RESPONSE_SCHEMA_NOT_IN_SPEC errors

nschonni committed:
chore: jsonfmt search (#6736)

MichalHel committed:
Michal dev security microsoft.security 2017 08 01 preview (#6563)

* changed enum name ruleType to RecommendationType in IoTSecuritySolution RP

* changed examples of IOT security solution RP

* nullable userDefinedResources query/subs

* fixing the Security Solution RP

* Changed the enums of the Recommendations type and status of the IOT Solution Manager RP

* changed property of the recommendation configuration form "description" to "name" in the IOT Solution Manager RP

* Fixed examples of IOT Security Manager RP

* Fixed error in schema related to IoT security Solution RP

* fixed enums in IoT Security solution RP

* fixed spelling mistakes in IoT solution Manager RP files

* Update specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/iotSecuritySolutions.json

Co-Authored-By: Nick Schonning <nschonni@gmail.com>

* Update specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/IoTSecuritySolutions/CreateIoTSecuritySolution_example.json

Co-Authored-By: Nick Schonning <nschonni@gmail.com>

* Fixed spelling mistakes in IoT solution manager RP

* Fixed spelling mistakes in IoT solution Manager RP

* removed unprintable characters and tabs from the IoT Security Solution json

Commit:
new tag in alerts management (#6689)

* new tag in alerts management

* try to fix errors

* more fixes

* merged action rule and ManagedResource to avoid error

* undo ManagedResource change, but removed required

* revert some changes that are not essential to fix linter

* added supressions

nschonni committed:
chore: jsonfmt security (#6747)

iscai-msft committed:
fixed datafactory readme, added java readme (#6431)

nschonni committed:
fix: logic model validation errors (#6751)

- Add "body" for RESPONSE_BODY_NOT_IN_EXAMPLE errors
- Change value to body for WorkflowTriggerHistories_Get
- Remove "description" from OperationList results

navysingla committed:
Nasingla/subscription text (#6339)

nschonni committed:
fix: Batch model validation errors (#6753)

- Add body for RESPONSE_BODY_NOT_IN_EXAMPLE errors
- active -> Active for state enum in 2017-09-01 examples

nschonni committed:
fix: applicationinsights delete examples (#6636)

nschonni committed:
fix: service-map model validation error (#6754)

Add "body" for RESPONSE_BODY_NOT_IN_EXAMPLE error

number213 commented:
@sergey-shandar could you please take a look?

xiaoyingQQ committed:
support dataset v2 split name (#6738)
